openChess.cpp

```diff
@@ -18,7 +17,7 @@ int main()
     std::string col;
     std::cout << "Pick your color (w / b): ";
     std::cin >> col;
-    g_playingAsWhite = col[0] == 'w' ? true : false;
```

Comment:
this was stupid to begin with:

`g_playingAsWhite = (col[0] == 'w')` would have been correct